### PR TITLE
minor fix for StabilityTest

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

File: integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/ConnectorKafkaSpec.scala [hunk 1]
```diff
@@ -34,6 +34,7 @@ class ConnectorKafkaSpec extends TestSpecBase {
   override def beforeAll(): Unit = {
     super.beforeAll()
     kafkaCluster.start()
+    Util.retryUntil(kafkaCluster.isAlive())
```
Comment (manuzhang) [inline integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/ConnectorKafkaSpec.scala]: if this is required for every test, better put it intot `kafkaCluster.start`

**Comment by stanleyxu2005** [comment]
+1

**Comment by stanleyxu2005** [comment]
@manuzhang good point. Let me make the change when I replace spotify/kafka with my more compacter kafka mini cluster in next PR

**Comment by manuzhang** [comment]
could you hold the new image till we have all tests ready ? Storm tests rely on kafka cluster and have only been run against the existing kafka image.  It's high risk to swap the image now.

**Pull request merged**